Lib9c/Action/ActionContextExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using Libplanet;
+using Libplanet.Action;
+
+namespace Nekoyume.Action
+{
+    public static class ActionContextExtensions
+    {
+        public static bool IsPreviewNet(this IActionContext context)
+        {
+            var goldCurrency = context.PreviousStates.GetGoldCurrency();
+            return goldCurrency.Minters
+                       .Contains(new Address("340f110b91d0577a9ae0ea69ce15269436f217da"))
+                   && goldCurrency.Ticker == "NCG"
+                   && goldCurrency.DecimalPlaces == 2;
+        }
+
+        public static bool IsMainNet(this IActionContext context)
+        {
+            var goldCurrency = context.PreviousStates.GetGoldCurrency();
+            return goldCurrency.Minters
+                       .Contains(new Address("47d082a115c63e7b58b1532d20e631538eafadde"))
+                   && goldCurrency.Ticker == "NCG"
+                   && goldCurrency.DecimalPlaces == 2;
+        }
```

Comment:
I suggest like this. and i think previewnet minter can be change because we can reset previewnet be required.

```
public enum NetworkType
{
    Mainnet,
    Previewnet,
    Internal
}

public NetworkType GetNetworkType(this IActionContext ctx)
{
    var gc = ctx.GetGoldCurrency();
    if (gc.Contains(mainnetAddress)
        return NetoworkType.Mainnet;
}
```

Reply:
Then, if it is assumed that the gold currency's minter can be changed, I guess this feature cannot be implemented. Right?

Reply:
I think, it would be helpful we can check `IsMainnet` right now.

Reply:
Then I'll remove `IsPreviewNet()` method. Thanks for your opinion.